mint.json

```diff
@@ -2,10 +2,10 @@
   "$schema": "https://mintlify.com/schema.json",
   "name": "Novu",
   "logo": {
-    "dark": "/logo/logo-dark-bg.svg",
-    "light": "/logo/logo-light-bg.svg"
+    "dark": "/logo/novu-logo-placeholder.svg",
+    "light": "/logo/novu-logo-placeholder.svg"
   },
-  "openapi": "https://api.novu.co/api-json",
+  "openapi": "https://api.novu.co/api-json?utm_campaign=docs-openapi",
```

Comment:
This one needs to be machine readable for the Mintlify crawler, I'll revert and update with the non-deprecated endpoint (https://api.novu.co/openapi.json).